packages/webpack/README.md

```diff
@@ -92,7 +92,25 @@ Using the experimental `--experimental-esbuild` option will replace the `babel-l
 
 To use it, install [`esbuild-loader`](https://www.npmjs.com/package/esbuild-loader) and [`esbuild-jest`](https://www.npmjs.com/package/esbuild-jest) as dev-dependencies in your project.
 
-Please report any bugs you may encounter.
+In order to use esbuild in Jest you need to set the `USE_EXPERIMENTAL_ESBUILD` environment variable to `true`.
+
+```sh
+USE_EXPERIMENTAL_ESBUILD=true yarn test
+```
+
+**Please be aware that things are working differently in esbuild than Babel. The currently known drawbacks and limitations are:**
+
+- esbuild is not on a stable release cycle yet. Please also read about the [production readyness](https://esbuild.github.io/faq/#production-readiness) from the main author.
+- esbuild does not typecheck your TypeScript files, it can only convert them to JavaScript.
+- to use the JSX syntax in TypeScript you have to use the `.tsx` extention. We recommend to also use the `.jsx` extention for JavaScript based JSX files to be consistent, even though it's not required.
+- Hops' `importComponent` is currently a Babel plugin and we only support the simple syntax (arrow function with import expression) in esbuild mode for now.
```

Comment:
I'd re-phrase to point out the reliance of `importComponent` on Babel. And that without Babel — but ESBuild — its feature set is diminished. And — most importantly and thus imho needs explicit mentioning — that it can only do default imports, but no named imports. This isn't clear from the code snippet for users, that aren't complete Hops pros.